### ISSUE TITLE
Time Zoon Update

### ISSUE THREAD

**Issue by yngpapers** [dillbyrne/random-agent-spoofer#476]
Please version 0.9.5.6, time zoon is no more available. Kindly please check to fix this error. Its really important. Thank you.

**Comment by lahm52** [comment]
Yes, the Time zone spoofing (UTC) is no longer available which i dont know why. It helps a lot cos the system time zone automatically change back to it default after changing it. Kindly please fix this and put it back.

**Comment by Wannabe-Programmer** [comment]
@yngpapers @lahm52 The time zone spoofing will be implemented soon. The reason why it's not longer available is because Dillbyrne had to re-write the script injection code to utilize frame scripts since Firefox versions past 44 broke script injection. 

He is still trying to find away to manipulate these requests from the context of using frame scripts. See #417 for more information and close this issue since it's a duplicate.

**Comment by KIMWW** [comment]
+1 Timezone spoofing
